### PR TITLE
Hibernate metamodel created

### PR DESCRIPTION
**Pull request by IvanBegar**
https://github.com/ita-social-projects/GreenCity/issues/1612

### CODE REVIEW

**Review by forestik — APPROVED** [review]

**Review by KhonkoDmytro — APPROVED** [review]

**Review by SashkoMolodec — APPROVED** [review]

File: service/src/main/java/greencity/service/HabitFactServiceImpl.java [hunk 1]
```diff
@@ -121,18 +124,20 @@ public HabitFactVO save(HabitFactPostDto fact) {
...
+        habitFact.setHabit(habit);
+        habitFact.getTranslations()
+            .forEach(habitFactTranslation -> {
+                habitFactTranslation.setContent(factDto.getTranslations().stream()
+                    .filter(newTranslation -> newTranslation.getLanguage().getCode()
+                        .equals(habitFactTranslation.getLanguage().getCode()))
+                    .findFirst().get()
+                    .getContent());
+                habitFactTranslation.setFactOfDayStatus(factDto.getTranslations().get(0).getFactOfDayStatus());
+            });
```
Comment (vterlyha) [inline service/src/main/java/greencity/service/HabitFactServiceImpl.java:139]: Move it to private method with proper name

File: dao/src/main/java/greencity/entity/HabitFact.java [hunk 2]
```diff
@@ -17,9 +17,9 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @OneToMany(cascade = {CascadeType.ALL}, mappedBy = "habitFact", fetch = FetchType.LAZY)
+    @OneToMany(cascade = {CascadeType.ALL}, mappedBy = "habitFact", fetch = FetchType.EAGER)
     private List<HabitFactTranslation> translations;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.EAGER)
```
Comment (vterlyha) [inline dao/src/main/java/greencity/entity/HabitFact.java:23]: Use EAGER only when every request in your application needs this filed. In other case use LAZY and if you need this field  when connection to DB is closed use custom query with inner join

File: service/src/main/java/greencity/service/HabitFactServiceImpl.java (hunk 1 above)
Comment (IvanBegar) [reply service/src/main/java/greencity/service/HabitFactServiceImpl.java:139]: done

File: dao/src/main/java/greencity/entity/HabitFact.java (hunk 2 above)
Comment (IvanBegar) [reply dao/src/main/java/greencity/entity/HabitFact.java:23]: fixed

**Pull request merged**